tests/providers/amazon/aws/transfers/test_sftp_to_s3.py

```diff
@@ -95,9 +85,10 @@ def test_sftp_to_s3_operation(self, use_temp_file=True):
         create_file_task.execute(None)
 
         # Test for creation of s3 bucket
+        s3_hook = S3Hook("aws_default")
```

Comment:
We do not need any connection here we have mock s3 anyway and better use as keyword
```suggestion
        s3_hook = S3Hook(aws_conn_id=None)
```